swift/StableDiffusion/pipeline/StableDiffusionPipeline.swift

```diff
@@ -114,12 +123,14 @@ public struct StableDiffusionPipeline: ResourceManaging {
     public func generateImages(
         prompt: String,
         negativePrompt: String = "",
+        startingImage: CGImage? = nil,
+        strength: Float = 1.0,
         imageCount: Int = 1,
         stepCount: Int = 50,
         seed: UInt32 = 0,
         guidanceScale: Float = 7.5,
         disableSafety: Bool = false,
-        scheduler: StableDiffusionScheduler = .pndmScheduler,
+        scheduler schedulerType: StableDiffusionScheduler = .pndmScheduler,
         progressHandler: (Progress) -> Bool = { _ in true }
```

Comment:
This method has too many parameters. We should consider using a `Configuration` struct.

Reply:
Created a struct called `SampleIn`